ls1intum/Artemis
# `Programming exercises`: Fix repositories download dialog with multiple selected exercise

### Checklist
#### General



### Motivation and Context


Currently, when exporting programming exercises repositories, the dialog is not completely displayed when selecting multiple exercises and therefore unusable. (see screenshot below)

### Description

The problem was due to an `*ngIf` checking the `isAtLeastInstructor` property of the single exercise to export, which is `undefined` when exporting multiple exercise together. Permissions are already checked before in this case, as the user has to be at least `instructor` within the course.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course with a Programming Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Select one or more programming exercises using the checkboxes on the left
4. Click the ![image](https://user-images.githubusercontent.com/45761921/195294759-aec4c69b-9a77-4fe4-8e43-99d387db3049.png) button and check if the dialog is complete
5. Navigate to the detail page of a programming exercise
6. Click the ![image](https://user-images.githubusercontent.com/45761921/195294759-aec4c69b-9a77-4fe4-8e43-99d387db3049.png) button and check if the dialog is complete


### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|


### Screenshots



#### Before
![image](https://user-images.githubusercontent.com/45761921/195292073-2be2d5a6-b5d2-42e4-8482-0dab4bdc5943.png)


#### After 
![image](https://user-images.githubusercontent.com/45761921/195292197-9ff58eeb-0727-4e4f-bf9c-1b1e765cd190.png)
